docs/Docker.md

```diff
@@ -45,5 +43,17 @@ docker run algorand/conduit init --importer algod --processors filter_processor
 With `conduit.yml` in your current working directory,
 launch the container:
 ```
-docker run -it -v $(pwd)/conduit.yml:/conduit/data/conduit.yml algorand/conduit
+docker run -it -v $(pwd)/conduit.yml:/data/conduit.yml algorand/conduit
 ```
+
+# Mounting the Data Directory
+
+The data directory is located at `/algod/data`. Mounting a volume at that location will allow you to resume the deployment from another container.
+
+## Volume Permissions
+
+The container executes in the context of the `algorand` user with UID=999 and GID=999 which is handled differently depending on your operating system or deployment platform. During startup the container temporarily runs as root in order to modify the permissions of /data. It then changes to the `algorand` user. This can sometimes cause problems, for example if your deployment platform doesn't allow containers to run as the root user.
```

Comment:
```suggestion
The container executes in the context of the `algorand` user with UID=999 and GID=999 which is handled differently depending on your operating system or deployment platform. During startup the container temporarily runs as root in order to modify the permissions of `/data`. It then changes to the `algorand` user. This can sometimes cause problems, for example if your deployment platform doesn't allow containers to run as the root user.
```